FreeRTOS-Plus/Demo/coreHTTP_Windows_Simulator/HTTP_S3_Download/DemoTasks/S3DownloadHTTPExample.c

```diff
@@ -214,15 +369,54 @@ static const char * pcRequestURI;
  */
 static void prvHTTPDemoTask( void * pvParameters );
 
+
+/**
+ * @brief Establish a HTTP connection with AWS S3 server.
+ *
+ * @param[in] pxTransportInterface The transport interface for making network calls.
```

Comment:
The name in comment for the function parameter `pxNetworkContext` does not match.